### ISSUE TITLE
Suggestion to make the impact of the problem clearer

### ISSUE THREAD

**Issue by jwbensley** [mxsasha/rpsl-registry-scoped-members#7]
[Issues 6](https://github.com/mxsasha/rpsl-registry-scoped-members/issues/6) discusses "what is going wrong" (ambiguity of members), but it's missing "what is the impact when that happens".

I think before moving on to the following statement, which is meant to justify the `src-members` field, it would be good to clarify the impact.

https://github.com/mxsasha/rpsl-registry-scoped-members/blob/a8983003b773a61cf6500bbe0e7d5c69c3ca4dbf/draft-romijn-rpsl-registry-scoped-members-00.md.md?plain=1#L57-L58

What do you think to something like the following?

>  The ambiguity encountered when resolving set members can result in either an incorrect RPSL object being chosen, because an object with the same primary key was pulled from the wrong IRR, or the required RPSL object (which does exist) is not found, because the resolving process didn't try to pull the object from the correct IRR.
> 
> With the former case, the result is that unintentional routing policy information maybe be computed and deployed to network infrastructure, which in the best case, leads to route leaking, and in the worst case, route hijacking. This has been seen multiple times on the public Internet.
> 
> With the later case, when routing policy information is missing, networks may loose their connectivity entirely.
> 
> There is no current way to prevent such ambiguity during set member resolution, both for operators who create the legitimate objects and those who try to resolve them. 

Optionally, you could provide a reference to Amazon AS-SET hijack: https://manrs.org/2022/12/why-network-operators-should-use-hierarchical-as-sets/

Optionally, you could provide a reference to AS-GOOGLE which exists in RIPE right now, and is empty / has no `members`: https://apps.db.ripe.net/db-web-ui/lookup?source=ripe&key=AS-GOOGLE&type=as-set, this is because in PDB we can see it should be read from RADB: https://www.peeringdb.com/net/433 which is not empty / has `members`.

**Comment by mxsasha** [comment]
Yes, good suggestion. I have added it, but slightly less decisive and a bit more "may happen" - same as in #9, it feels like links in the draft don't entirely fit.
